Update ISOs.md (#318)
MicrosoftDocs/windows-insider: wip/ISOs.md

> 2. Connect a USB drive that's at least 8GB. **Back up any data on the drive you want to save, because any data on it will erased.**

Reviewer: ```suggestion
2. Connect a USB drive that's at least 8 GB. **Back up any data on the drive you want to save, because any data on it will be erased.**
```

Author: Sure 👍

- If you use this drive to do a clean Windows install, connect it to the target device and reboot. You may also need to adjust your BIOS or UEFI settings to allow it to boot from a USB-based drive.

> ## Clean installation with a Windows 11 ISO (Not an Insider Preview build)

Reviewer: ```suggestion
## Clean installation with a Windows 11 ISO (Not an Insider Preview Build)
```

Author: Sure 👍

1. Back up all your files and applications. A clean install with a Windows 11 ISO will erase all of your files, settings, and applications.

2. [Download the latest Windows 11 ISO](https://www.microsoft.com/software-download/windows11) by saving it to a location on your PC or creating a bootable USB.

3. Open File Explorer and go to where you saved the ISO. Select the ISO file to open it, which lets Windows mount the image so you can access it.

4. Select the setup.exe file to launch the install process.

> 5. During setup, select the **Change what to keep** option, and then choose **Nothing** on the next screen to complete a clean intall.

Reviewer: ```suggestion
5. During setup, select the **Change what to keep** option, and then choose **Nothing** on the next screen to complete a clean install.
```

Reviewer: The same correction needs to be made to line 107, please.

Author: Sure 👍
